application_code/client_apps/java/debezium_replicator_producer_consumer/src/main/java/com/example/pulsarworkshop/PulsarConsumerFullCfg.java

```diff
@@ -0,0 +1,186 @@
+package com.example.pulsarworkshop;
```

Comment:
Please use a different file name other than `PulsarConsumerFullCfg.java` that is more relevant with your use case.

Reply:
Good catch. This was an oversight.